fixups/RegLegacyFixups/RegistryFixups.cpp

```diff
@@ -564,13 +600,15 @@ LSTATUS __stdcall RegOpenKeyExFixup(
     LARGE_INTEGER TickStart, TickEnd;
     QueryPerformanceCounter(&TickStart);
     DWORD RegLocalInstance = ++g_RegIntceptInstance;
-
     auto entry = LogFunctionEntry();
 
     Log("[%d] RegOpenKeyEx:\n", RegLocalInstance);
+    auto [updatedSubKey, keypath] = ReplaceRegistryQueryPath(&key, subKey);
+    if (updatedSubKey)
+    {
+        return RegOpenKeyExImpl(key, updatedSubKey, options, samDesired, resultKey);
+    }
 
-
-    std::string keypath = InterpretKeyPath(key) + "\\" + InterpretStringA(subKey);
     std::string registryPath = ReplaceRegistrySyntax(keypath);
```

Comment:
This is covered now in ReplaceRegistryQueryPath, can we avoid this duplicate call ?

Reply:
Updated